ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -11,18 +9,37 @@ function createInstallJunit() {
   EXIT_CODE_CLUSTER=6
   if test -f "${SHARED_DIR}/install-status.txt"
   then
-    EXIT_CODE=`cat ${SHARED_DIR}/install-status.txt | awk '{print $1}'`
-    cp "${SHARED_DIR}/install-status.txt" ${ARTIFACT_DIR}/
+    EXIT_CODE=`tail -n1 "${SHARED_DIR}/install-status.txt" | awk '{print $1}'`
+    cp "${SHARED_DIR}/install-status.txt" "${ARTIFACT_DIR}/"
     if [ "$EXIT_CODE" ==  0  ]
     then
+      set +o errexit
+      grep -q "^$EXIT_CODE_INFRA$" "${SHARED_DIR}/install-status.txt"
+      PREVIOUS_INFRA_FAILURE=$((1-$?))
+      set -o errexit
+
       cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
-      <testsuite name="cluster install" tests="6" failures="0">
+      <testsuite name="cluster install" tests="6" failures="$PREVIOUS_INFRA_FAILURE">
```

Comment:
does `tests="6"` need to be scaled by attempt counts too?

Reply:
Ah good call, thanks